src/conduit/components/comment.cljs

```diff
@@ -14,12 +14,16 @@
       [:img.comment-author-img {:src avatar-url}]
       (base/Button "Post Comment")]]))
 
-(rum/defc Options []
-  [:div.mod-options
-   (base/Icon :edit)
-   (base/Icon :trash-a)])
+(rum/defc Options < rum/reactive [r comment]
+  (let [user (rum/react (citrus/subscription r [:user]))
+        author-username (get-in comment [:author :username])
+        {params :route-params} (rum/react (citrus/subscription r [:router]))
+        handle-delete #(citrus/dispatch! r :comments :delete-comment (:id params) (comment :id) (user :token))]
```

Comment:
not important but in general `(get map key)` or `(key map)` is faster than `(map key)`